WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -214,13 +218,28 @@ def create
     permitted_params = [
       :guests,
       :comments,
+      :accepted_at,
+      :deleted_at,
       registration_competition_events_attributes: [:id, :competition_event_id, :_destroy],
     ]
+    params[:registration][:deleted_at] = nil
+    params[:registration][:accepted_at] = nil
     if current_user.can_manage_competition?(competition_from_params)
-      permitted_params << :accepted_at
+      permitted_params << [
+        :accepted_by,
+        :deleted_by,
+      ]
```

Comment:
Are you sure this does what you want? I think this will add an array of length 2 inside of the existing `permitted_params` array:

```ruby
irb(main):001:0> arr = []
=> []
irb(main):002:0> arr << 3
=> [3]
irb(main):003:0> arr << [ 3, 4 ]
=> [3, [3, 4]]
```